src/schema/lastmod_test.go

```diff
@@ -151,3 +151,29 @@ func TestManifestRead(t *testing.T) {
 		}
 	}
 }
+
+func TestManifestChange(t *testing.T) {
+	var corpus = _m(t)
+	corpus.build()
+	corpus.write()
+	var cwd, _ = os.Getwd()
+	var fname = filepath.Join(cwd, "testdata", "foo.dat")
+
+	var pre = _m(t)
+	pre.build()
+	if !corpus.equiv(pre) {
+		t.Fatalf("Pre-create, manifests should be the same")
+	}
+
+	var err = os.WriteFile(fname, []byte("foo"), 0644)
+	if err != nil {
+		t.Fatalf("Unable to write file %q: %s", fname, err)
+	}
+	defer os.Remove(fname)
```

Comment:
Defer is really handy, but it can't be guaranteed - say if the app exits (crash or otherwise) or something. Super irritating in some cases.

Reply:
Oh, yeah that could be a mess. Though it would be the same if the code were written with the commands manually written at the end of the code block etc. So those concerns aren't really `defer`'s problem. Would have to use signal trapping for handling anything with a crash, yes?